MapboxVision/Services/MotionManager.swift

```diff
@@ -24,8 +24,9 @@ final class MotionManager {
         motion.showsDeviceMovementDisplay = true
         
         motion.startDeviceMotionUpdates(using: referenceFrame, to: queue) { [weak self] (data, error) in
-            guard let data = data else { return }
-            self?.handler?(data)
+            guard let data = data, let referenceFrame = self?.motion.attitudeReferenceFrame else { return }
```

Comment:
Do we need a local state?
Can we use `func startDeviceMotionUpdates(to: OperationQueue, withHandler: CMDeviceMotionHandler)`?

Reply:
decided to not change